eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -26,13 +26,20 @@ __Required:__ No
 __Description:__  A URI where further information can be obtained, if applicable.
 
 ## Legal Link Types
+
+### COMPOSITION
+__Required:__ Yes  
+__Legal targets:__ [EiffelCompositionDefinedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelCompositionDefinedEvent.md)  
+__Multiple allowed:__ No  
+__Description:__ Identifies the composition from which an artifact declared by [EiffelArtifactCreatedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelArtifactCreatedEvent.md) was built, or the composition for which an already existing artifact is declared reused by [EiffelArtifactReusedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelArtifactReusedEvent.md).
```

Comment:
Description does not talk about need of link between EiffelCompositionDefinedEvent and EiffelArtifactDeployedEvent

Reply:
1 - The COMPOSITION link is there to make possible to reference more than one artifact 
2-  The COMPOSITION text explanation is described in the context of the EiffelCompositionDefinedEvent (e.g. on the same page).  The the core Eiffel 2.0 events the links are in a generic page, and in that context more explanation would have to be given

Reply:
So all the artifacts which will be deployed will be made in to a composition prior to deployment Sorry I could not get clear picture here?
The COMPOSITION Text in links.md page, It should be used as part of EiffelArtifactReusedEvent, If we are planning to resuse a artifact in building the composition since it is not resbuilt. and point to the old composition.
It can be used in EiffelArtifactCreatedEvent if we are building a kind of single artifact for entire composition like a zip to identify the composition., 

So Use Case involving composition event could better explain this. Because in the use case we never mentioned about EiffelCompositionDefinedEvent.